share/search/fetchers.py

```diff
@@ -210,64 +218,132 @@ class CreativeWorkFetcher(Fetcher):
             , 'subject_synonyms', COALESCE(subjects.synonyms, '{}')
             , 'related_agents', COALESCE(related_agents, '{}')
             , 'retractions', COALESCE(retractions, '{}')
+            , 'lineage', COALESCE(lineage, '{}')
         ) AS _source
         FROM share_creativework AS creativework
-             LEFT JOIN LATERAL (SELECT sources FROM all_sources WHERE creativework_id = creativework.id) AS sources ON TRUE
-             LEFT JOIN LATERAL (SELECT tags FROM all_tags WHERE creativework_id = creativework.id) AS tags ON TRUE
-        LEFT JOIN LATERAL (SELECT related_agents FROM all_related_agents WHERE creativework_id = creativework.id) AS related_agents ON TRUE
+
         LEFT JOIN LATERAL (
-                    SELECT array_agg(identifier.uri) AS identifiers
-                    FROM share_workidentifier AS identifier
-                    WHERE identifier.creative_work_id = creativework.id
-                    ) AS links ON TRUE
+            SELECT sources FROM all_sources WHERE creativework_id = creativework.id
```

Comment:
can you add more comments to explain the code

Reply:
I added some comments to help explain and break it up into more manageable chunks.